jaxrs/src/main/java/com/cerner/beadledom/jaxrs/JaxRsParamConditions.java

```diff
@@ -43,10 +43,9 @@ private JaxRsParamConditions() {
   public static void checkParam(boolean expression, Object errorMessage) {
```

Comment:
Im not sure why it's an Object instead of a `String`.  Changing that now, would make it a non-passive change. 

Should this method be added to the docs?